### PR TITLE
[feat] Direct Search Item Component

### PR DESCRIPTION
**Pull request by jjstnlee**
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Created a new component for project items in the default modal list thingy
- When you click on the listing it will open up the corresponding project's modal :3
- Added small energy type icons in assets/icons
- Added new Heading2 in texts.ts to use for project name in project item list thing
- Added new color code for navy 0.75 opacity in colors.ts
- Changed TagText2 in texts.ts to use new color code for navy 0.75 opacity

### Screenshots
[//]: # "Add any screenshots that will help your PR reviewer understand what’s going on in the PR!"
[//]: # "[Required] For frontend tasks: add screenshots of the layout and expected behavior (GIFs if you’re feeling extra fancy)!"
[//]: # "Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

https://github.com/user-attachments/assets/4fa5c6b9-2d8f-4e99-8123-9161625f068d

image

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
- Check out new ProjectItem folder in components folder
- Run pnpm dev and check localhost:3000/testing page to play around w button and see if it works

## Next steps
[//]: # "[Optional] - What has not yet been added to the PR, doesn’t yet have any functionality — essentially, what needs to be fixed or added."
[//]: # "[Optional] any temporary fixes in this PR that should be cleaned up later."
- ~~font is not the same as figma for some reason, idk if its my fault or if the fonts in the style sheet isn't working or something~~ i fixed the fonts i think but the project size and energy type looks a little darker/thicker than on the figma
- ~~need to change symbol for whether its operational or in progress (rn its just a green dot for everything)~~ fixed
- need to change it so that when project name is too long it overflows into the next line

### Online sources
[//]: # "[Optional] insert links to any useful tutorials or documentation that you used for this PR!"

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @itsliterallymonique

### CODE REVIEW

File: assets/Icons/icons.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
+export const CheckmarkIcon = () => (
```
Comment (itsliterallymonique) [inline assets/Icons/icons.tsx:1]: I think you should keep a KDM icon folder so all icons are organized into their own folders

File: components/ProjectItem/index.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import Image from 'next/image';
+import {
+  queryDefaultImages,
+  queryProjectbyId,
+} from '@/api/supabase/queries/query';
+import {
```
Comment (itsliterallymonique) [inline components/ProjectItem/index.tsx:9]: pull from main + rebase then use the icons that Neha has imported. Check her PR for how she added the colors for the renewable energy technology icons: https://github.com/calblueprint/ace-ny/pull/63

You can also use the new components she made for renewable energy technology and status (if it works for your sprint -- I am not sure if the fonts / text are the same)

**Review by itsliterallymonique — CHANGES_REQUESTED** [review]
Looks good. just need to pull + rebase then use the icons Neha added.

**Review by itsliterallymonique — CHANGES_REQUESTED** [review]
Looks good julee. you can push + merge
